src/main/java/com/google/devtools/build/lib/starlarkbuildapi/StarlarkActionFactoryApi.java

```diff
@@ -345,7 +345,14 @@ void write(FileApi output, Object content, Boolean isExecutable)
                     + "<p>The content of this file (generally one of the outputs of the action) "
                     + "corresponds to  the list of input files that were not used during the whole "
                     + "action execution. Any change in those files must not affect in any way the "
-                    + "outputs of the action."),
+                    + "outputs of the action."
+                    + ""
+                    + "<p>When using this mechanism, consider adding an entry to"
+                    + "<code>execution_requirements</code> with key"
+                    + "<code>internal-inline-outputs</code> and value the path of this file to "
+                    + "provide remote execution services with a hint that this file will be read "
+                    + "directly after the action execution and should be inlined into the action "
+                    + "result."),
```

Comment:
This is not needed, as it happens automatically here: https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/analysis/actions/StarlarkAction.java;l=152;drc=4c4bbec9336eb8b943879a77bb782f771d21ab7a. I will remove it on import.